tutorial/westeros/westeros_baseline_using_xlsx_import_part1.ipynb

```diff
@@ -0,0 +1,294 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Westeros Tutorial Part 4.a - Building a scenario by importing data from Excel"
```

Comment:
Is this really 4.a.? A glance at the list in tutorials/README.rst suggests it's something like (2)(8)(1).

Perhaps better to omit the numbers entirely, so they don't have to be adjusted for every addition/removal/change of order.

Reply:
@khaeru and @behnam-zakeri I have listed below the current numbering scheme across tutorials. It seems there are some issues. It would be good to as @khaeru suggested to either remove or correct all numbering. 
The numbering though can provide users some guidance as to an order in which to approach features, also indicating how complexity may increase.  I have suggested an alternative numbering in the second table below across all tutorials - numbering added in (). Either way we should do this either as part of this or a separate PR to clean this up for the next release.
Part 1 - baseline
Part 2a.i - Emission bounds
Part 2a.ii - Emission taxes
Part 2b - renewables - firm capacity (PR #368 suggests this to change to 3a.)
Part 2c - renewables - flex. generation (PR #369 suggests this to change to 3b.)
Part 2d - seasonality 
Part 3 - Reporting
Part 3a. - fossil resources
Part 3b. - Share constraints
Part 3c. - Addon technologies
Part 3c. - Renewable supply curves (see PR #370 )
Part 4a. - xlsx import part 1
Part 4b. - xlsx import part 2

--------------
Part 1 (1)- baseline
Part 2a.i (2a) - Emission bounds
Part 2a.ii (2b) - Emission taxes
Part 3a. (3) - fossil resources
Part 3b. (4) - Share constraints
Part 3c. (5) - Addon technologies
Part 2b (6a) - renewables - firm capacity (PR #368 suggests this to change to 3a.)
Part 2c (6b) - renewables - flex. generation (PR #369 suggests this to change to 3b.)
Part 3c. (6c) - Renewable supply curves (see PR #370 )
Part 2d (7) - seasonality 
Part 3 (8) - Reporting
Part 4a. (9a) - xlsx import part 1
Part 4b. (9b) - xlsx import part 2

Reply:
@khaeru I will remove all the numbering throughout all the tutorials, except for the three which will be updated via PR #368 #369 #370
As discussed with @behnam-zakeri the issue that we should address though is to provide better guidance throughout the tutorial ensemble. I will make an issue for this and we can update this in the future, so as to not hinder this PR from getting merged now.

Reply:
All titles have been adapted: see below

Westeros Tutorial - Introduction to the MESSAGEix framework: Creating a baseline scenario 
Westeros Tutorial - Introducing emissions (part1/2): Adding emission bounds
Westeros Tutorial - Introducing emissions (part2/2): Adding emission taxes
Westeros Tutorial - Introducing resource: Adding fossil resources
Westeros Tutorial - Introducing share constraints
Westeros Tutorial - Introducing `addon` technologies
Westeros Tutorial - Introducing sub-annual time-steps: Modeling of variability in energy supply and demand
Westeros Tutorial - Introducing reporting
Westeros Tutorial - Introducing the use of Excel files: Building a scenario by importing data from a single ".xlsx" file
Westeros Tutorial - Introducing the use of Excel files: Building a scenario by importing data from multiple ".xlsx" file